doc/source/changes/version_0_21.rst.inc

```diff
@@ -0,0 +1,110 @@
+﻿New features
+------------
+
+* implemented replaced_axes() method to replace one, several or all axes of an array
+  (instead of using the workaround a.with_axes(a.axes.replace(...)))
+  (closes :issue:`67`).
```

Comment:
Previously, to replace some axes but not all, one had to use a.with_axes(a.axes.replace(...))